django/db/models/query.py

```diff
@@ -470,6 +470,37 @@ def bulk_create(self, objs, batch_size=None):
 
         return objs
 
+    def bulk_save(self, objs, fields, batch_size=None):
```

Comment:
Since this only works for instances with an pk, do you think that `bulk_update` would be a better name? The regular `save()` method can either create or update depending on pk status which may confuse users here.

Reply:
I considered this, but `queryset.update()` is the best 'bulk update' method. I didn't want to confuse the two, this is more about saving multiple model fields with multiple differing values, gene `bulk_save`. Open to changing it though.